contributing/development/handling_compatibility_breakages.rst

```diff
@@ -0,0 +1,110 @@
+.. _doc_handling_compatibility_breakages:
+
+Handling compatibility breakages
+=================================
+
+So you've added a new argument to a method, changed the return type,
+changed the type of an argument, or changed the default value of an argument,
+and now the automated testing is complaining about compatibility breakages?
```

Comment:
Technically those are parameters (declared in the signature) rather than arguments (values passed in). But I've seen Godot often use the latter... so not sure?